README.md

```diff
@@ -32,9 +32,9 @@ Differing from the current official [Nix](https://github.com/NixOS/nix) installe
 * In `nix.conf`:
   + the `auto-allocate-uids`, `nix-command` and `flakes` features are enabled
```

Comment:
Here too
```suggestion
  + the `nix-command`, `flakes`, and `auto-allocate-uids` (On Linux only) features are enabled
```

Reply:
We do actually leave the feature enabled on Mac...

Reply:
.......ah